libs/decoupling/helmet/project.json

```diff
@@ -3,7 +3,7 @@
   "$schema": "../../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "libs/decoupling/helmet/src",
   "prefix": "lib",
-  "tags": [],
+  "tags": ["type:hlm"],
```

Comment:
question: What is this for ?

Reply:
I wasn't seeing the linting error that was described in the challenge:
```
If you go to [BtnHelmetDirective](https://angular-challenges.vercel.app/challenges/libs/decoupling/helmet/src/lib/btn-style.directive.ts), you will encounter a linting error. A project tagged with type:hlm can only depend on libs tagged with type:core.
```

I added this so that I would see the linting error.

Reply:
oh that's true, I forgot it. I need to update the challenge 👍